### DISCUSSION TITLE
Pro Self Hosted - Mac Intel - Multiple RustDesk Icons In Dock

### DISCUSSION THREAD

**Discussion by risaw1981** [rustdesk/rustdesk#9316] [Q&A] [answered]
Hey gang,

I have a self hosted Pro RustDesk setup, only a few Mac users who are all intel based and seem to have the same issue. when you restart or shutdown the Mac and have enabled RustDesk to start-up as a service, it will add a new icon to the dock every time you restart the Mac. I currently now have 3 RustDesk icons in my Dock. I can close them but that's not the point :)

They are running on 1.2.7 custom build apps

Any ideas on this,

**Comment by rustdesk** [comment, marked as answer]
https://github.com/rustdesk/rustdesk/issues/9088

**Reply by risaw1981** [reply]
Can I use the nightly build on a self hosted instance?

**Reply by rustdesk** [reply]
It is already in current custom client (1.3.0)

**Reply by risaw1981** [reply]
OK thanks, Ill regenerate the installer and install.

**Reply by rustdesk** [reply]
Please confirm if current nightly works for you first.

**Reply by risaw1981** [reply]
Problem fixed! Thanks, both Nightly and custom 1.3.0 fixed the issue.
